deploy/openshift/install-openshift.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/sh
```

Comment:
I think the name of this script should change. "install-openshift.sh" makes it seem like you are installing openshift itself. Rather, this script should be called something like "deploy-openshift-yaml.sh" or "deploy-kiali-in-openshift.sh", or something like that.